BB-SC/MWLayer/Communication/BB_Smart_Charging_Communication.md

```diff
@@ -0,0 +1,77 @@
+
+# Smart Charging Communication
+
+## BB Tags(s)
+<!-- Tag(s) define in which area(s) (cloud, in-vehicle) the BB is executed, and what type of BB it is (tool, process, microservice) -->
+BB-SC
+
+## Functional Clusters
+<!-- In which Functional Cluster the BB be located; if none of the existing fit new required -->
+Communication
+
+## Layer
+<!-- AppLayer, MWLayer, OSLayer, HWLayer -->
+MWLayer
+
+## Known Implementation
+
```

Comment:
https://lfenergy.org/projects/everest/

Reply:
Agreed. Will be taken care of in the next pull request.